mlir-tensorrt/executor/lib/Conversion/MemRefToExecutor.cpp

```diff
@@ -548,6 +550,43 @@ void executor::populateMemRefToExecutorPatterns(
 }
 
 namespace {
+
+class RemoveNoOpClonePattern : public OpRewritePattern<bufferization::CloneOp> {
```

Comment:
This is illegal. Clone indicates creating a copy, which you are not doing here. This indicates something else is missing in the pipeline before this point.

Reply:
@christopherbate Let me know if you any suggestion on how to fix this.